e2e/scripts/check-airgap-installation-state.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+set -euox pipefail
+
+wait_for_installation() {
+    ready=$(kubectl get installations --no-headers | grep -c "Installed" || true)
+    counter=0
+    while [ "$ready" -lt "1" ]; do
+        if [ "$counter" -gt 36 ]; then
+            echo "installation did not become ready"
+            kubectl get installations 2>&1 || true
+            kubectl describe installations 2>&1 || true
+            kubectl get charts -A
+            kubectl describe chart -n kube-system k0s-addon-chart-ingress-nginx
+            kubectl get secrets -A
+            kubectl describe clusterconfig -A
+            echo "operator logs:"
+            kubectl logs -n embedded-cluster -l app.kubernetes.io/name=embedded-cluster-operator --tail=100
+            return 1
+        fi
+        sleep 5
+        counter=$((counter+1))
+        echo "Waiting for installation"
+        ready=$(kubectl get installations --no-headers | grep -c "Installed" || true)
+        kubectl get installations 2>&1 || true
+    done
+}
+
+wait_for_nginx_pods() {
+    ready=$(kubectl get pods -n kotsadm -o jsonpath='{.items[*].metadata.name} {.items[*].status.phase}' | grep "nginx" | grep -c Running || true)
+    counter=0
+    while [ "$ready" -lt "1" ]; do
+        if [ "$counter" -gt 36 ]; then
+            echo "nginx pods did not appear"
+            kubectl get pods -n kotsadm -o jsonpath='{.items[*].metadata.name} {.items[*].status.phase}'
+            kubectl get pods -n kotsadm
+            kubectl logs -n kotsadm -l app=kotsadm
+            return 1
+        fi
+        sleep 5
+        counter=$((counter+1))
+        echo "Waiting for nginx pods"
+        ready=$(kubectl get pods -n kotsadm -o jsonpath='{.items[*].metadata.name} {.items[*].status.phase}' | grep "nginx" | grep -c Running || true)
+        kubectl get pods -n nginx 2>&1 || true
+        echo "ready: $ready"
+    done
+}
+
+ensure_app_deployed() {
```

Comment:
copied this script from `check-installation-state.sh` and changed this function so that it does not use the KOTS CLI